### PR TITLE
Support  ZGC  add normal gc phrase

### PR DESCRIPTION
**Pull request by xu1009**

### CODE REVIEW

File: language-agent/JVMMetric.proto [hunk 1]
```diff
@@ -83,6 +83,7 @@ message GC {
 enum GCPhrase {
     NEW = 0;
     OLD = 1;
+    NORMAL = 2;
```
Comment (wu-sheng) [inline language-agent/JVMMetric.proto:86]: Please add comments about what is a normal GC phase.

Comment (xu1009) [reply language-agent/JVMMetric.proto:86]: where should i comment，near this code 

like 
 NEW = 0;
    OLD = 1;
    NORMAL = 2;  // comments

Comment (wu-sheng) [reply language-agent/JVMMetric.proto:86]: It is fine either like your style or this style.

```suggestion
    // ....
    NORMAL = 2;
```

Comment (xu1009) [reply language-agent/JVMMetric.proto]: ok

**Review by wu-sheng — APPROVED** [review]

**Comment by BFergerson** [comment]
@wu-sheng, I think `GCPhrase` is supposed to be `GCPhase`. A "phrase" is a set of words.
Instead it looks like it should be phase, like: https://www.baeldung.com/jvm-zgc-garbage-collector#gc-phase-properties

---

Would you like me to provide PRs for this project and skywalking-java?

**Comment by wu-sheng** [comment]
I think this is a legacy design. We tried to make the GC concept easier than official. So, we group them as young GC and old GC. And this is why normal is added as no young and old.

What do you suggest? @BFergerson We also need to consider back-forward compatibility.

**Comment by BFergerson** [comment]
I'm not very familiar with Protobuf, but it seems to be possible to do per https://stackoverflow.com/questions/58801521/protobuf-message-enum-type-rename-and-wire-compatibility.

**Comment by wu-sheng** [comment]
I think the name doesn't value. Index(0, 1, 2) and type(int or object) matter only.
I used to serialize an object to binary, then deserialize to another object. Once they are logically matching from type perspective, it is good to go.

So, the key question is, what do you suggest to change? Which phases are you suggesting to add? And we need to check whether it makes sense for OAL scripts and UI. 
This back-forward compatibility is not just about protobuf, it is also for agent and OAP server version mismatch. Such as `8.8.0 agent -> 9.0.0 OAP(next release)` or `8.9.9 agent(next release) -> 8.9.1 OAP`.

**Comment by BFergerson** [comment]
I'm not suggesting adding anything. I just noticed a grammar error in the enum name, that's all. If you Google `"gcphrase"`, you only get by SkyWalking links. If you google `"gcphase"`, you get back results from other projects.

At this point, it might not be worth the headache to fix. I just noticed this PR and was confused about what a "GC phrase" was.

**Comment by wu-sheng** [comment]
Oh, I see. I think we could change w/o breaking anything. The SkyWalking-java has `latest->OAP 8.9.x` testing to verify whether break. I could change and try.

**Comment by wu-sheng** [comment]
See https://github.com/apache/skywalking-data-collect-protocol/pull/60

**Pull request merged**